Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -326,7 +310,7 @@ void doPublishOdom() {
   double vth = (vr - vl)/(2 * TRACK_RADIUS);
   
   // Calculate the deltas since
-  double dt = current_time.toSec() - last_odom_publish_time.toSec();
+  double dt = current_time.toSec() - last_raw_odom_publish_time.toSec();
```

Comment:
But I guess this is a function, not a class, which is only used in one case, not used by many.
You initialize a `static` variable, as you did below.
```
static double last_raw_odom_publish_time(ros_nh.now());
```
Initialization of `static` variables only happens once.